daemon/api.go

```diff
@@ -1901,13 +1901,18 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	if err != nil {
 		return InternalError("cannot get user count: %s", err)
 	}
+
 	if len(users) > 0 && !createData.ForceManaged {
 		return BadRequest("cannot create user: device already managed")
 	}
 
 	// special case: the user requested the creation of all known
 	// system-users
 	if createData.Email == "" && createData.Known {
+		if release.OnClassic && !createData.ForceManaged {
```

Comment:
I think we want this next to the `len(users) > 0` check above instead of under this branch, as the condition we're checking for is virtually the same: the device has users.